pom.xml

```diff
@@ -57,6 +57,7 @@
                   <directory>${project.build.directory}/ozone</directory>
                   <excludes>
                     <exclude>distro/configs/openmrs/frontend_config/ozone_logo*.svg</exclude>
+                    <exclude>distro/configs/openmrs/initializer_config/globalproperties/02_ozone-distro/disable-demo-patients.xml</exclude>
```

Comment:
I am not sure if we should do something like what you have done or do it like

```
distro/configs/openmrs/initializer_config/globalproperties/**/disable-demo-patients.xml
```

WDYT?

Reply:
Yeah. I will do similarly in LAMP, HSC and Haiti